src/ChipInput.spec.js

```diff
@@ -512,12 +512,12 @@ describe('blurBehavior modes', () => {
     jest.runAllTimers()
 
     expect(tree.find('input').getDOMNode().value).toBe('')
-    expect(setTimeout).toHaveBeenCalledTimes(0)
+    /* expect(setTimeout).toHaveBeenCalledTimes(0) */
```

Comment:
I'm not sure if we even need to test how often `setTimeout` was called. If there was a timeout before adding the chip, the next `expect` should fail because we didn't run it in the meantime.

Reply:
So, should we delete this line then?